db/models/network.model.js

```diff
@@ -121,12 +121,6 @@ class Network extends Model {
       sourceKey: "id"
     });
 
-    this.hasMany(models.pullRequest, {

```

Comment:
but we have many deliverables, instead, where's that definition? [we need it on the model as well]

Reply:
we take it from the bounty model using associations